src/EditorFeatures/Core.Wpf/InlineRename/AbstractEditorInlineRenameService.SymbolRenameInfo.cs

```diff
@@ -146,24 +149,21 @@ public TextSpan GetReferenceEditSpan(InlineRenameLocation location, Cancellation
                 return new TextSpan(location.TextSpan.Start + position, replacementText.Length);
             }
 
+            private string GetWithoutAttributeSuffix(string value) => value.GetWithoutAttributeSuffix(isCaseSensitive: _isCaseSensitiveAttribute);
+
             private static string GetSpanText(Document document, TextSpan triggerSpan, CancellationToken cancellationToken)
             {
                 var sourceText = document.GetTextAsync(cancellationToken).WaitAndGetResult(cancellationToken);
                 var triggerText = sourceText.ToString(triggerSpan);
                 return triggerText;
             }
 
-            private static string GetWithoutAttributeSuffix(string value)
-            {
-                return value.GetWithoutAttributeSuffix(isCaseSensitive: true);
-            }
-
             internal bool IsRenamingAttributeTypeWithAttributeSuffix()
             {
                 if (this.RenameSymbol.IsAttribute() || (this.RenameSymbol.Kind == SymbolKind.Alias && ((IAliasSymbol)this.RenameSymbol).Target.IsAttribute()))
                 {
-                    var name = this.RenameSymbol.Name;
-                    if (name.TryGetWithoutAttributeSuffix(isCaseSensitive: true, result: out name))
+                    var name = GetWithoutAttributeSuffix(this.RenameSymbol.Name);
```

Comment:
Sorry I don't think I expressed my idea clearly last time. Instead of changing to not use the  `TryGetWithoutAttributeSuffix` method, I meant to say that a local `TryGetWithoutAttributeSuffix` helper could use the case sensitive flag a reduce the number of times we had to specify it. I think it would be good to change this back to using the Try method.